interactions/api/models/channel.py

```diff
@@ -1126,6 +1147,43 @@ async def get_webhooks(self) -> List[Webhook]:
         res = await self._client.get_channel_webhooks(int(self.id))
         return [Webhook(**_, _client=self._client) for _ in res]
 
+    async def get_members(self) -> List[ThreadMember]:
+        """
+        Gets the list of thread members
+
+        :return: The members of the thread.
+        :rtype: List[ThreadMember]
+        """
+        if not self._client:
+            raise LibraryException(code=13)
+        if not self.thread_metadata:
+            raise LibraryException(message="The Channel you specified is not a thread!", code=12)
+
+        res = await self._client.list_thread_members(int(self.id))
+        return [ThreadMember(**member, _client=self._client) for member in res]
+
+    async def leave_thread(self) -> None:
```

Comment:
```suggestion
    async def leave(self) -> None:
```
Remove redundant naming